app/controllers/idv/in_person/state_id_controller.rb

```diff
@@ -130,7 +130,7 @@ def copy_state_id_address_to_residential_address(pii_from_user)
       end
 
       def updating_state_id?
-        pii_from_user.has_key?(:first_name)
+        pii_from_user.has_key?(:first_name) && user_session.dig(:idv, :ssn).present?
```

Comment:
I'm not sure I see this issue in main when I use the FSM version of this update route. Do you know what's changed between the controller version and the FSM version that is leading to this bug?

Reply:
i think the difference is the form is being submitted earlier